.semaphore.yml

```diff
@@ -0,0 +1,37 @@
+# Setting for Semaphore CI
```

Comment:
As you don't have plan to use semaphoreCI, I think we should remove all semaphore related stuff